docs/using-the-jdbc-driver/HostAvailabilityStrategy.md

```diff
@@ -0,0 +1,23 @@
+# Host Availability Strategy
+
+## What is Host Availability Strategy
+The AWS JDBC Driver keeps track of the availability of database hosts. A host's availability status can be either `AVAILABLE` or `NOT_AVAILABLE`. This information is used by the driver when deciding which hosts to connect to.  
```

Comment:
I wonder if we should rename `NOT_AVAILABLE` to `UNAVAILABLE`

Reply:
I dont feel too strongly either way, but I think Im more used to seeing `UNAVAILABLE` being used.  
Sergiy mentioned `UNAVAILABLE` can look similar to `AVAILABLE`, so `NOT_AVAILABLE` is easier to read when quickly scanning the code. But  he asked to leave the decision in our hands. 
Do you feel particularly strongly either way?

Reply:
I just went ahead and renamed it. Done!